test/quaternions.jl

```diff
@@ -1,3 +1,10 @@
+struct Degree{T} <: Number
```

Comment:
```suggestion
struct Degree{T} <: Real
```

Reply:
I'm not sure about adding this type to Makie directly... would it be possible to leave it to another package and make things work via ducktyping?

Reply:
Oh, didn't see this is just for testing purposes... Excellent, so I guess it's already like I requested